site/src/Librecores/ProjectRepoBundle/Entity/ProjectClassification.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A ProjectClassification contains the classifications for the projects
+ * A group up related categories can be used to classify the projects
+ *
+ * @author Sandip Kumar Bhuyan <sandipbhuyan@gmail.com>
+ *
+ * @ORM\Table(name="ProjectClassification")
+ * @ORM\Entity
+ */
+class ProjectClassification
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="id", type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * It contains the classification string which has been classified by
+     * facted classification. Classifications are categorization of a project
+     * and two consecutive categories will be separated by '::'
+     *
+     * @var string
+     *
+     * @ORM\Column(name="categories", type="text")
+     */
+    private $classification;
+
+    /**
+     * It contains the project id and related to Project Entity in
+     * One Project can have Many ProjectClassification
+     *
+     * @var Project
+     *
+     * @ORM\ManyToOne(targetEntity="Project", inversedBy = "classifications")
+     * @ORM\JoinColumn(nullable=false, onDelete="CASCADE")
+     */
+    private $project;
+
+    /**
+     * It conatins the creation date of the classification
```

Comment:
typo, and remove the "It contains the". Simply "Creation date/time" is sufficient.